misc/libtool.m4

```diff
@@ -1033,7 +1033,7 @@ if test -n "$_LT_AC_TAGVAR(hardcode_libdir_flag_spec, $1)" || \
    test -n "$_LT_AC_TAGVAR(runpath_var, $1)" || \
    test "X$_LT_AC_TAGVAR(hardcode_automatic, $1)" = "Xyes" ; then
 
-  # We can hardcode non-existant directories.
+  # We can hardcode non-existent directories.
```

Comment:
The typos in this file are also from an upstream project, but they've since fixed it.  I'm actually not sure why we have misc/libtool.m4 when we also have m4/libtool.m4 (which doesn't have this typo).  Maybe this copy of the file could just go away.